swagger.yaml

```diff
@@ -386,6 +386,32 @@ paths:
           description: "Invalid request body or unexpected Cognito response"
           schema:
             $ref: '#/definitions/ErrorResponse'
+    get:
+      tags:
+        - Groups
+      summary: "Returns a list of users in a given group"
+      description: "Returns a list of users in a given group in Cognito"
+      produces:
+        - "application/json"
+      parameters:
+        - in: path
+          name: name
+          type: string
+          required: true
+          description: the group's name
+      responses:
+        200:
+          description: "User added to group"
```

Comment:
Looks like this description has been copied from the post above, just wants to be updated to be relevant to this endpoint

Reply:
Thank you for spotting this, I have updated this.